openapi/src/parts/controlled_aws_bucket.yaml

```diff
@@ -112,6 +112,27 @@ components:
           description: A valid bucket location per https://docs.aws.amazon.com/general/latest/gr/s3.html.
           type: string
 
```

Comment:
We don't yet know what the lifecycle rules are for AWS, so this might be a bit early.  AWS doesn't have a concept of "default storage class", for instance; specifying storage class must be done at object create time, which is "data path" functionality and not intermediated by Terra.

Reply:
I really think we should leave this out for now, as we cannot meaningfully influence the storage class for AWS buckets in the same way we can for GCP buckets.  We will be able to influence lifecycle, but again this will most likely diverge from what we do with GCP semantically.  Lets just not implement this until we have a clearer picture on what we will do on the lifecycle front.